com.stansassets.scene-management/BuildConfigurator/Editor/EditorNBuildSettingsDrawer.cs

```diff
@@ -0,0 +1,77 @@
+using StansAssets.Plugins.Editor;
+using UnityEditor;
+using UnityEngine;
+
+namespace StansAssets.SceneManagement.Build
+{
+    internal class EditorNBuildSettingsDrawer
+    {
+        internal void DrawSettings()
+        {
+            using (new IMGUIBlockWithIndent(new GUIContent("Editor & Build Settings")))
+            {
+                PreventingDialogs();
+                DrawDuplicates();
+                DrawScenesSync();
+            }
+        }
+
+        void PreventingDialogs()
+        {
+            EditorGUIUtility.labelWidth = 300f;
```

Comment:
Should be drop 'labelWidth' to where it was after we draw what we need?
Feel like this can break some UI or be very frustrating to debug.
Alos would be more to use 'using' for ths